server/app/lib/api_analytics_tracer.rb

```diff
@@ -0,0 +1,17 @@
+module ApiAnalyticsTracer
+  API_LOGGER = Logger.new(File.join(Rails.root, 'log', 'api-queries.log'))
+  def analyze_query(query: )
+    query_type = query.selected_operation.selections&.first&.name
+
+    return unless query_type == '__schema'
```

Comment:
I think this needs to be inverted right? We skip analytics if it is an introspection query.

Reply:
Ugh, thanks, I always goof these